sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveContext.scala

```diff
@@ -95,12 +95,12 @@ class HiveContext(sc: SparkContext) extends SQLContext(sc) {
    */
   def refreshTable(tableName: String): Unit = {
     // TODO: Database support...
-    catalog.refreshTable("default", tableName)
+    catalog.refreshTable(Seq("default", tableName))
   }
 
   protected[hive] def invalidateTable(tableName: String): Unit = {
     // TODO: Database support...
-    catalog.invalidateTable("default", tableName)
+    catalog.invalidateTable(Seq("default", tableName))
```

Comment:
Why hardcode default here?  When a database isn't specified its up to the catalog to use the current one, where as this will ignore any `USE` statements that have been run.